### PR TITLE
refactor: refactored docs in LendingPool.sol

### LINKED ISSUE #148: Protocol 2.5 - Refactor LendingPool documentation
Move all the documentation for LendingPool to the interface and use inheritdoc in the lendingpool contract

### PR DESCRIPTION
**Pull request by The-3D**
closes #148

### CODE REVIEW

File: contracts/interfaces/ILendingPool.sol [hunk 1]
```diff
@@ -395,20 +395,47 @@ interface ILendingPool {
...
     );
 
+  /**
+   * @dev Initializes a reserve, activating it, assigning an aToken and debt tokens and an
+   * interest rate strategy
+   * - Only callable by the LendingPoolConfigurator contract
+   * @param asset The address of the underlying asset of the reserve
+   * @param aTokenAddress The address of the aToken that will be assigned to the reserve
+   * @param stableDebtAddress The address of the StableDebtToken that will be assigned to the reserve
+   * @param aTokenAddress The address of the VariableDebtToken that will be assigned to the reserve
```
Comment (dhadrien) [inline contracts/interfaces/ILendingPool.sol:405]: should be @param `variableDebtAddress`

File: contracts/interfaces/ILendingPool.sol [hunk 2]
```diff
@@ -395,20 +395,47 @@ interface ILendingPool {
...
+  /**
+   * @dev Sets the configuration bitmap of the reserve as a whole
+   * - Only callable by the LendingPoolConfigurator contract
+   * @param asset The address of the underlying asset of the reserve
+   * @param configuration The new configuration bitmap
+   **/
+  function setConfiguration(address asset, uint256 configuration) external;
 
   /**
    * @dev Returns the configuration of the reserve
```
Comment (dhadrien) [inline contracts/interfaces/ILendingPool.sol:441]: Returns the state and configuration of the reserve

File: contracts/interfaces/ILendingPool.sol [hunk 3]
```diff
@@ -451,37 +478,97 @@ interface ILendingPool {
...
   function setPause(bool val) external;
 
+  /**
+   * @dev Returns if the LendingPool is paused
+   */
   function paused() external view returns (bool);
 
+  /**
+   * @dev Authorizes/Unauthorizes a flash borrower. Authorized borrowers pay no flash loan premium
+   * @param flashBorrower address of the flash borrower
```
Comment (dhadrien) [inline contracts/interfaces/ILendingPool.sol:524]: could add the `* - Only callable by the LendingPoolConfigurator contract`

File: contracts/interfaces/ILendingPool.sol [hunk 4]
```diff
@@ -451,37 +478,97 @@ interface ILendingPool {
...
+  /**
+   * @dev Returns whether a flashborrower is authorized (pays no premium)
+   * @param flashBorrower address of the flash borrower
+   * @return `true` if authorized, `false` if not
+   */
   function isFlashBorrowerAuthorized(address flashBorrower) external view returns (bool);
 
+  /**
+   * @dev Updates flash loan premiums
+   * flash loan premium consist in 2 parts
```
Comment (dhadrien) [inline contracts/interfaces/ILendingPool.sol:538]: could add the `* - Only callable by the LendingPoolConfigurator contract`

**Review by dhadrien — APPROVED** [review]

**Review by ernesto-usal — APPROVED** [review]

**Pull request merged**